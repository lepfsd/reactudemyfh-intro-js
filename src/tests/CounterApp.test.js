import { shallow } from 'enzyme';
import CounterApp from '../CounterApp';

describe('prueba en <CounterApp />', () => {

    test('debe mostrar el mensaje', () => {

        const wrapper = shallow( <CounterApp />);
        expect(wrapper).toMatchSnapshot()

    });

    test('debe mostrar 100', () => {

        const wrapper = shallow( <CounterApp value={100} />);
        const counterText = wrapper.find('h2').text().trim()

        expect( counterText ).toBe('100')


    });
});