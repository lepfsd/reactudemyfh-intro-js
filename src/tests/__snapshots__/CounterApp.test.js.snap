// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prueba en <CounterApp /> debe mostrar el mensaje 1`] = `
<Fragment>
  <h1>
    CounterApp
  </h1>
  <h2>
     
    0
     
  </h2>
  <button
    onClick={[Function]}
  >
    +1
  </button>
  <button
    onClick={[Function]}
  >
    -1
  </button>
  <button
    onClick={[Function]}
  >
    reset
  </button>
</Fragment>
`;
